src/biome/text/modules/heads/token_classification.py

```diff
@@ -31,6 +46,14 @@ def __init__(
         feedforward: Optional[FeedForwardConfiguration] = None,
     ) -> None:
         super(TokenClassification, self).__init__(backbone)
+
+        self._label_encoding = label_encoding
+
+        # Convert span labels to tag labels if necessary
+        # We just check if "O" is in the label list, a necessary tag for IOB/BIOUL schemes, an unlikely label for spans
+        if "O" not in labels:
```

Comment:
what happens if we have a label "O" for spans. Even if unlikely, can we test for this (both here and with a test in unit tests? can we warn you have "O" but not other IOB/BIOUL looking labels?

Reply:
> what happens if we have a label "O" for spans. Even if unlikely, can we test for this (both here and with a test in unit tests?

If you provide a label called "O" for spans, it will fail while training, because it will not find the "B-O" or "I-O" tags in the label vocabulary. Should i just test that this fails or do you want to support that use case? Maybe a specific parameter switch in the TaskHead, like `label_type = "spans"/"tags"`?

>  can we warn you have "O" but not other IOB/BIOUL looking labels?
Sure, i added a warning in that case.

Reply:
Great! No need to support this use case or add `label_type`